CmrUI/Screens/NewsFeedTab/NewsFeedVM.swift

```diff
@@ -0,0 +1,57 @@
+                ,
+                     feedType: FeedType.feedText),
+            FeedItem(image: UIImage(named: Constants().kImgName + "\(5)"), title: "Fifth title", subtitle: "Fifth subtitle", content: "Fifth content", feedType: FeedType.feedImage)
+        ]
+        
+        dataSource.sort(by: sorterTime)
+    }
+    
+    func sorterTime(this: FeedItem, that: FeedItem) -> Bool {
```

Comment:
@vmsavka Yes, but the FeedItems have no time at all yet, and the title is still hard coded as WEDNESDAY xxxxx

Reply:
fixed